app/src/main/java/org/jellyfin/androidtv/ui/home/HomeFragment.kt

```diff
@@ -106,6 +109,28 @@ class HomeFragment : StdBrowseFragment(), AudioEventListener {
 			// Update the views before creating rows
 			views = callApi<ItemsResult> { apiClient.GetUserViews(TvApp.getApplication().currentUser!!.id, it) }
 
+			// Check for live TV support
+			if (TvApp.getApplication().currentUser!!.policy.enableLiveTvAccess) {
```

Comment:
Not really necessary but you could check if there is at least 1 home section for live tv before doing the api call.